prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -75,7 +75,7 @@ class FakeReport {
                         else -> TODO()
                     }
                 }
-                Element.Type.TELEPHONE -> faker.phoneNumber().cellPhone()
+                Element.Type.TELEPHONE -> faker.numerify("##########") // faker.phoneNumber().cellPhone()
```

Comment:
Look's fine, but being flexible on the format may be needed.

Reply:
check